modulefiles/ufs_noaacloud.intel.lua

```diff
@@ -0,0 +1,20 @@
+help([[
+loads UFS Model prerequisites for cloud/intel
+]])
+
+prepend_path("MODULE_PATH", "/contrib/spack-stack/envs/ufs-wm/install/modulefiles/Core")
```

Comment:
has this stack been fully tested for the weather model RTs on each CSP? no issue with cpld_control_p8? if not, i think it is best to hold on moving forward with spack based modules.

Reply:
@ulmononian I have not tested any spack installations. Sounds like @mark-a-potts has completed his installations. I'd be happy to test them.

Reply:
okay, just wanted to check because @gspetro-NOAA is working on the docs and i think those modulefiles were causing some issues.